const.go

```diff
@@ -101,25 +101,27 @@ var ConfToEnv = map[string]string{
 
 	// [Clients]
 
-	// [Clients.Command]
-	"clients.command.port": "CLIENTS_COMMAND_PORT",
+	// FIXME: these are wrong, should be core-command, core-data, and core-metadata, support-*
+	// [Clients.core-command]
+	"clients.core-command.port": "CLIENTS_CORECOMMAND_PORT",
 
-	// [Clients.CoreData]
-	"clients.coredata.port": "CLIENTS_COREDATA_PORT",
+	// [Clients.core-data]
+	"clients.core-data.port": "CLIENTS_COREDATA_PORT",
 
+	// FIXME: was this fixed for Ireland???
```

Comment:
It seems so.All the go services are using clients.core-data and clients.core-metadata. However, bacnet-c and grove-c are using clients.data. It does look like that is just still to be updated - device-sdk-c itself has been updated. So I think it is safe to remove this.